vxquery-xtest/src/main/java/org/apache/vxquery/xtest/XTestOptions.java

```diff
@@ -52,4 +52,7 @@
 
     @Option(name = "-htmlreport", required = false, usage = "HTML Report output file")
     String htmlReport;
-}
\ No newline at end of file
+    
+    @Option(name = "-hdfs", required = false, usage = "run HDFS tests")
+    boolean hdfs;
```

Comment:
You can add specific test for hdfs. Since this is new code, add enough tests to cover all use cases.